app/src/main/java/com/courseproject/tindar/ui/home/ViewProfileFragment.java

```diff
@@ -182,36 +183,25 @@ public void onDestroyView() {
         binding = null;
     }
 
-    /**
-     * Downloads image from link.
-     */
-    private class DownloadImage extends AsyncTask<String, Void, Bitmap> {
-        ImageView profilePicture;
-
-        public DownloadImage(ImageView profilePicture) {
-            this.profilePicture = profilePicture;
-        }
-
-        protected Bitmap doInBackground(String... urls) {
-            String urldisplay = urls[0];
-            Bitmap mIcon11 = null;
-            try {
-                InputStream in = new java.net.URL(urldisplay).openStream();
-                mIcon11 = BitmapFactory.decodeStream(in);
-            } catch (Exception e) {
-            }
-            return mIcon11;
-        }
-
-        protected void onPostExecute(Bitmap result) {
-            profilePicture.setImageBitmap(result);
-        }
-    }
-
     private void setsProfile() {
         ViewProfileResponseModel profile =
                 viewProfilesController.getProfile(otherUserId);
 
+        profileLink = profile.getProfilePictureLink();
+        if (profileLink.equals("spongebob1.png")){
```

Comment:
can you comment on it right above this line something like
// This is only a temporary implementation for demo purposes. The plan is to connect to show the picture from the external URL.

Reply:
I wonder if a Switch statement would be good for this case

Reply:
i tried it but it wasn't loading the images sometimes so i switched it back to the if else